packages/swingset-liveslots/src/virtualObjectManager.js

```diff
@@ -996,7 +1015,9 @@ export const makeVirtualObjectManager = (
             // objects, but they could invoke all their equivalent methods,
             // by using e.g.
             // cohort1.facetA.foo.apply(cohort2.facetB, [...args])
-            Number(facet) === index || Fail`illegal cross-facet access`;
+            if (Number(facet) !== index) {
+              return undefined;
+            }
```

Comment:
@FUDCo , this is a semantic change to existing virtual kind functionality. Is it still correct?

Reply:
I checked, and only my own code consumes this, which I know is prepared for these `undefined`s. So it cannot be an incorrect change based on code I don't know. Resolving.